libmscore/mscore.cpp

```diff
@@ -164,6 +164,7 @@ void Direction::fillComboBox(QComboBox* cb)
       }
 
 static Spatium doubleToSpatium(double d)       { return Spatium(d); }
+static Direction intToDirection(int i)         { return Direction(i); }
```

Comment:
This fix makes me wonder... we use value<T> a little bit everywhere in the code. Somehow we probably don't use it these code paths much because they wouldn't work...